packages/graphback-datasync/src/deltaHelper.ts

```diff
@@ -0,0 +1,46 @@
+import { ModelDefinition, getTableName } from '@graphback/core';
+import { Db, Collection, ObjectId } from "mongodb";
+import { DataSyncFieldNames } from './util';
+
+export function getDeltaTableName(tableName: string) {
+  return `${tableName}Delta`
+}
+
+/**
+ * M
+ */
```

Comment:
comments looks incomplete here.. can we extend this a little bit. 

It is useful since we use `typedoc` to generate API reference (even though this is not part of something we'll want to expose but let's get the habit running).